wls-gui-wahllokalsystem/src/components/common/inputs/BaseDateInput.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-text-field
+    :model-value="formatedDate"
+    type="date"
+    :rules="[REQUIRED]"
+    label="Datum"
+    clearable
+    @update:model-value="onModelValueChanged"
+    @clear="onClearClicked"
```

Comment:
nur mit `@clear` wird die funktion bei mir nicht getriggert
```suggestion
    @click:clear="onClearClicked"
```

Reply:
clear-Handler entfernt